src/database/UIDatabase.js

```diff
@@ -175,6 +179,11 @@ class UIDatabase {
   write(...args) {
     return this.database.write(...args);
   }
+
+  getSetting(key) {
+    const setting = this.database.get('Setting', key, 'key');
+    return setting && setting.value;
```

Comment:
mmm maybe this should be `(setting && setting.value || '')`

Reply:
I'm not sure about usage and the expectations of callers.. so will defer to whatever you think best. I like having a default myself